src/main/java/io/github/sskorol/model/Category.java

```diff
@@ -0,0 +1,25 @@
+package io.github.sskorol.model;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+/**
+ * Enum for available categories.
+ */
+@Getter
+@RequiredArgsConstructor
+public enum Category {
+
+    BRAND("Brand"),
+    AMAZON_FASHION(""),
+    AMAZON_PRIME(""),
+    SELLER("Seller"),
+    SHOE_SIZE("Shoe Size"),
+    SHOE_WIDTH("Shoe Width"),
+    COLOR("Color"),
+    WOMEN_FRAGRANCE_SIZE("Women's Fragrance Size"),
+    MOVIE_TV("Movie & TV Show Release Decade"),
+    TOYS_AGE_RANGE("Toys Age Range");
+
+    private final String category;
```

Comment:
it might be confusing to give to your property the same name as for class...
let's say you're calling `Category.BRAND.getCategory()`... first and last parts duplicate each other...